### PR TITLE
Automatically download datacats/lessc as needed

### PR DESCRIPTION
**Pull request by JackMc**
Also allow specifying a --devel option to 'datacats pull' to download all images.

### CODE REVIEW

File: datacats/cli/pull.py [hunk 1]
```diff
@@ -22,9 +25,16 @@ def pull(opts):
     """Download or update required datacats docker images
 
 Usage:
-  datacats pull
+  datacats pull [-d]
+
+Options:
+  -d --devel         Pull extra "development" images. These
```
Comment (wardi) [inline datacats/cli/pull.py]: I like `--all`
"Pull optional images as well as required images. Optional images will be pulled when needed, ..."

Comment (JackMc) [inline datacats/cli/pull.py]: Okay :)

File: datacats/cli/less.py [hunk 2]
```diff
@@ -7,5 +16,17 @@ def less(environment, opts):
...
+        for status in pull_stream(LESSC_IMAGE):
+            if 'status' not in status:
+                raise DatacatsError('Docker error: ' +
+                        (status['error'] if 'error' in status else 'Unknown error.'))
+            else:
+                sys.stdout.write('.')
+                sys.stdout.flush()
+        sys.stdout.write('\n')
+        sys.stdout.flush()
+
```
Comment (wardi) [inline datacats/cli/less.py]: don't duplicate this code, make a function that the pull command uses and call it instead

**Comment by dborzov** [comment]
Woah, I had no idea datacats even wraps less parser.

Lazy loading of `datacats/less` looks neat though!  :+1:

**Pull request merged**